README.md

```diff
@@ -87,17 +87,17 @@ Every post is structured with two main sections - the `meta` and `content` secti
 #### Example of blog post meta section
 
 ```yaml
-title: This is hello world post!
+title: 'This is hello world post!'
 author: anirrudh-krishnan
 published: October 14, 2022
-description: Lorem ipsum dolor sit amet
+description: 'Lorem ipsum dolor sit amet'
```

Comment:
Nope. I kinda assumed it was some search engine or link preview.

Reply:
Honestly, I haven't even looked at them yet. Maybe they have some snippet that will be useful to someone at some point?! All are blog are migrated from another system and these were made for this one so I figured it could be useful to have them around.